src/app/public/modules/progress-indicator/progress-indicator.component.ts

```diff
@@ -69,6 +71,10 @@ export class SkyProgressIndicatorComponent implements AfterContentInit, OnDestro
   private _isPassive: boolean;
   private ngUnsubscribe = new Subject();
 
+  constructor(
+    @Optional() private windowSvc: SkyWindowRefService
```

Comment:
Need to avoid abbreviations for variables at all costs, please. `windowService`. We can talk about "why" in our standup.

Reply:
alright